staff/lab/mod-printer

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+"""Remove or add printers to/from CUPS classes."""
+import argparse
+import subprocess
+import sys
+
+
+def modify_printer(args):
+    """Perform internal call to CUPS CLI to modify class(es)."""
+    # Determine which CUPS classes are available
+    cups_classes = subprocess.check_output(["lpstat", "-p"]).decode("utf-8").split("\n")
+    # Remove empty trailing newline
+    cups_classes.remove("")
+    for i in range(len(cups_classes)):
+        clazz = cups_classes[i]
+        # Output is of the form: printer <printername> is idle. <etc>
+        clazz = clazz[len("printer "):]
+        clazz = clazz[:clazz.index(" is ")]
+        cups_classes[i] = clazz
+
+    if args.verbose:
+        print("Found CUPS classes: " + str(cups_classes))
+    
+    args_clazz = getattr(args, "class")
+    if args_clazz and args_clazz in cups_classes:
```

Comment:
If `args_clazz` isn't in the class list, this code will act as if the arg wasn't specified, when in reality it means the user probably made a mistake, which could lead to unintended consequences. Instead, if it's not in the class list, return an error early and tell the user it doesn't exist.